CMakeLists.txt

```diff
@@ -9,7 +9,7 @@
 cmake_minimum_required(VERSION 3.19.6...3.29)
 
 if(POLICY CMP0157)
-  cmake_policy(SET CMP0157 NEW)
+  cmake_policy(SET CMP0157 OLD)
```

Comment:
What was the reason for changing this policy?

Reply:
Ah, that isn't mean to be part of this change. I have that changed locally as it prevents building for Android when building on Windows.